charts/incubator/alist/values.yaml

```diff
@@ -0,0 +1,18 @@
+image:
+  repository: tccr.io/truecharts/alist
+  pullPolicy: IfNotPresent
+  tag: latest
+
+env: {}
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 5244
+        port: 5244
```

Comment:
If possible I prefer continueing in our current 101** range of ports, instead of 5244.
The lower-the-port, the higher the chance of future conflict and for unimportant apps like this, that is annoying.